consensus/src/state_computer.rs

```diff
@@ -224,6 +229,9 @@ impl StateComputer for ExecutionProxy {
                 block_executor_onchain_config,
             )
             .await;
+        observe_block(timestamp, BlockStage::EXECUTION_PIPELINE_INSERTED);
```

Comment:
Not sure if there is any value in adding this - this is literally sending the command over the channel.

Reply:
This is the time between the block creation to inserting the block in the execution pipeline. Just wanted to see if this duration is very different from the time it takes for the block to get ordered.